03-task-manager/starter/controller/tasks.js

```diff
@@ -0,0 +1,98 @@
+const Task = require('../models/task')
+const getAllTasks = async (req, res) => {
+    // res.send('get All tasks')
+    try {
+        const tasks = await Task.find({});
+        res.status(200).json({tasks:tasks});
```

Comment:
You can use short-hand object notation and omit the value in an object if it is the same name as the key.

```javascript
{ tasks } // this is the same as { tasks: tasks }
```

Reply:
Understood! Corrected.